Packages/Ink/Editor/Tools/File Icons/InkBrowserIcons.cs

```diff
@@ -111,8 +111,8 @@ static InkBrowserIcons() {
 	    static void OnDrawProjectWindowItem(string guid, Rect rect) {
 	        string path = AssetDatabase.GUIDToAssetPath(guid);
 			if (InkEditorUtils.IsInkFile(path)) {
-				DefaultAsset asset = AssetDatabase.LoadAssetAtPath<DefaultAsset>(path);
-				DrawInkFile(InkLibrary.GetInkFileWithFile(asset), rect);
+				InkFile asset = AssetDatabase.LoadAssetAtPath<InkFile>(path);
+				DrawInkFile(asset, rect);
```

Comment:
InkFile is now a Scriptable Object.
No `OnDrawProjectWindowItem` would be needed if we assign a custom icon to InkFile.cs from inspector